spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -484,7 +484,14 @@ def certificate_valid?
       # @return (Bool) Is the current provisioning profile valid?
       #                To also verify the certificate call certificate_valid?
       def valid?
-        return status == 'Active'
+        # Provisioning profiles are not invalidated automatically on the dev portal when the certificate expires.
```

Comment:
Does this mean that this PR also fixes the fact that we were not detecting expired certificates properly before?

Reply:
Yes 😅. I don't know why Apple does not revoke profiles or make them invalid.
But even in this case, we cannot sign an app since the certificate has expired.

The date is Dec 4, and the expiration date is Dec 1.
The only Expired cert is that I opened it to see why it's not expired, and it expired immediately after opening it.
![image](https://github.com/fastlane/fastlane/assets/3704581/d2fd59b3-63db-489b-b81f-569899d647a4)

Reply:
They didn't bother setting up like a cron job that goes and expire everything like once a day 😂